src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,69 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+  a:hover {
+   background-color: yellow;
+   fill:yellow;
+ }
+
+ svg{
+    max-height: 106px;
+    max-width: fit-content;
+    margin-top: 94px;
+    background-color: white;
+ }
+  
+ .display-svgs-border-match {
+    max-height: 381px;
+    border: outset;
+    border-color: green;
+}
+
+ .display-original-optimised{
+    overflow-y: scroll;
+    max-height: 571px;
+    border: outset;
+    border-color: green;
+ }
+
+ .button-background{
+    background-color: grey;
+    height: 171px;   
+ }
+
+ .loader-text{
+      margin-top: 210px;
+ }
+
+ .loader {
+   border: 16px solid greenyellow;
+    border-radius: 114%;
+    border-top: 16px solid #bcda12; 
+    width: 197px;
+    height: 199px;
+    animation: spin 4s linear infinite;
+   margin-top: 100px;
+    margin-left: 856px;
+ }
+
+ /* Safari */
+@-webkit-keyframes spin {
+   0% { -webkit-transform: rotate(0deg); }
+   100% { -webkit-transform: rotate(360deg); }
+ }
+ 
+ @keyframes spin {
+   0% { transform: rotate(0deg); }
+   100% { transform: rotate(360deg); }
+ }
+
+ li{
+     display: table-cell;
+     margin: 5px; padding: 10px; 
+     vertical-align: top; 
+     position:relative; 
+     text-align: center;
+     left: 226%
```

Comment:
this would be bad practice

Reply:
So for instance, how would you define what left 226% does. Its obscure and less meaningful and could influence future styling in a negetive way. just like how we use padding instead of margin to prevent unintended influence on styling done outside of our component. we wouldnt use left as it's scope could refer to a global property or a div above etc, additionally 226 isnt very descriptive as what is it 226% of. best practice is relative based on your experience and the experience of others you work with. there is no clear cut good and bad. it is usually based on discretion, and if you ask someone why they would like something to be done in  a specific way and you feel their response makes sense, then you can adapt it to your record of best practice, if not you can dispute it and argue why another technique might be better

Reply:
Apparently, I have used another way to place my button at the center of the screen because that what I was trying to achieve but 50% was failing and I increased that % up until it was in the position that I desired.